sdk/include/opentelemetry/sdk/trace/simple_processor.h

```diff
@@ -14,7 +14,7 @@ namespace trace
  *
  * OnEnd and ForceFlush are no-ops.
  */
-class SimpleSpanProcessor : public SpanProcessor
+class SimpleSpanProcessor : public ForkAwareSpanProcessor
```

Comment:
This doesn't seem right to be added to the `SimpleSpanProcessor`. Can this be implemented in its own separate class? For example, we are not explicitly covering support for Windows 10 Universal Platform `App::OnSuspending` (platform -specific feature) in a **Simple** processor. I think *Simple* span processor has to remain simple.

Reply:
Please see the [comment](https://github.com/open-telemetry/opentelemetry-cpp/pull/274#discussion_r469422028) above